packages/airnode-node/src/evm/authorization/authorization-fetching.ts

```diff
@@ -8,23 +8,24 @@ import { go } from '@api3/promise-utils';
 import { ApiCall, AuthorizationByRequestId, Request, LogsData } from '../../types';
 import { CONVENIENCE_BATCH_SIZE, DEFAULT_RETRY_TIMEOUT_MS } from '../../constants';
 import { AirnodeRrpV0, AirnodeRrpV0Factory } from '../contracts';
+import { ChainAuthorizers } from '../../config';
 
 export interface FetchOptions {
-  readonly authorizers: string[];
+  readonly authorizers: ChainAuthorizers;
   readonly airnodeAddress: string;
   readonly airnodeRrpAddress: string;
   readonly provider: ethers.providers.JsonRpcProvider;
 }
 
 export async function fetchAuthorizationStatus(
   airnodeRrp: AirnodeRrpV0,
-  authorizers: string[],
+  authorizers: ChainAuthorizers,
   airnodeAddress: string,
   apiCall: Request<ApiCall>
 ): Promise<LogsData<boolean | null>> {
   const contractCall = (): Promise<boolean> =>
     airnodeRrp.checkAuthorizationStatus(
-      authorizers,
+      authorizers.requesterEndpointAuthorizers,
```

Comment:
I assumed here that this contract call is currently supposed to take in the `requesterEndpointAuthorizations`, so please let me know if I should have been mapping all authorizer contractts from the new authorizers object (values) and passing them in here.